docs/DeveloperGuide.md

```diff
@@ -1,29 +1,115 @@
+```DeleteListCommand``` is used to delete a whole packing list in the BagPacker application.
+
+Mechanism: ```DeleteListCommand.execute()``` reassigns the existing ```packingList``` to a new empty ArrayList of Items, thus deleting the ```packingList```.
+
 
 
 ## Product scope
+
 ### Target user profile
 
-{Describe the target user profile}
+BagPacker is for NUS students, in particular, exchange students who travel a lot and want a simple CLI to keep track of their packing.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+BagPacker aims to help busy students simplify their packing process by allowing easy adding of items to pack and record of the items they have already packed so that they can be organised and aboard their travels with ease.
+
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ... | I want to ...                                  | So that I can ...                                                            |
+|---------|----------|------------------------------------------------|------------------------------------------------------------------------------|
+| v1      | user     | Add an item to my packing list                 | update my packing list                                                       |
+| v1      | user     | Remove an item from my packing list            | update my packing list                                                       |
+| v1      | user     | view a list of my packed and unpacked items    | keep track of my packing list                                                |
+| v1      | user     | mark an item as packed                         | keep track of what is packed                                                 |
+| v1      | user     | mark an item as unpacked                       | keep track of what is unpacked                                               |
+| v1      | new user | see usage instructions                         | refer to them when I forget how to use the application                       |
+| v2a     | user     | find an item by name                           | find the pack status of an item without having to go through the entire list |
+| v2a     | user     | remove my packing list                         | clear my list once I am done packing                                         |
+| v2a     | user     | Specify the quantity of an item I need to pack | keep track of individual item quantities being packed                        |
+| v2b     | user     | Save my packing list                           | keep track of my packing list even after leaving the app                     |
 
 ## Non-Functional Requirements
 
```

Comment:
should be better to add in instructions for manual testing to get an overview of all commands?